Source/SwiftLintFramework/Models/RuleIdentifier.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  RuleIdentifier.swift
+//  SwiftLint
+//
+//  Created by Frederick Pietschmann on 3/5/18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+import Foundation
+
+public enum RuleIdentifier: Hashable, ExpressibleByStringLiteral {
+    case all
+    case some(identifier: String)
```

Comment:
what do you think about using `ruleID` instead of `some`? 🤔  

Sorry for not catching this earlier.

Reply:
`.some` is actually some legacy naming when the enum was still designed with cases `all` and `.some(let identifiers)` (there were many), so it doesn't fit properly now that it only holds one identifier. Yet I wouldn't go to with `ruleID`which would be confusing to use (`RuleIdentifier.ruleID`). Additionally `ruleID` rather describes the associated value than the case itself.

Since `.some` nonetheless doesn't fit well, I suggest using `case explicit(identifier: String)`